### PR TITLE
Hope to list a new protocol called Arctic

### PR DESCRIPTION
**Pull request by LpcAries**
Hope to list a new protocol called Arctic
---
##### Name (to be shown on DefiLlama):
Arctic

##### Twitter Link:
https://twitter.com/Arctic_trade

##### List of audit links if any:
https://docsend.com/view/42a2ixu3ey6zu3nx

##### Website Link:
https://arctic.trade/trade/swap

##### Logo (High resolution, preferably in .svg and .png, for application on both white and black backgrounds. Will be shown with rounded borders):
https://drive.google.com/file/d/1jx1rPbB6MZpOdtkI8ENHcJSOVVK2cz70/view?usp=sharing

##### Current TVL:
$104.01k

##### Chain:
Aurora Chian;

##### Short Description (to be shown on DefiLlama):
Powered by iZUMi Finance, Arctic is leading the next chapter of Aurora’s development as the core ecosystem-DEX, bringing DeFi innovation to Aurora in the forms of the veNFT and DL-AMM (Discretized Liquidity-AMM).

##### Token address and ticker if any:
ARC on Aurora: 0xBCD4e5E349Cc2810830E94f7fFEa4BD6b04c5978

##### Category (full list at https://defillama.com/categories) *Please choose only one:
Dexes

##### Oracle used (Chainlink/Band/API3/TWAP or any other that you are using):
NA

##### forkedFrom (Does your project originate from another project):
iZiSwap

##### methodology (what is being counted as tvl, how is tvl being calculated):
Summing up the token values of all pools.

### CODE REVIEW

**Comment by llamatester** [comment]
The arctic adapter exports: 
        
 
 _______________________________________
    Volume for 11/10/2022
    _______________________________________
    
    AURORA 👇
    Backfill start time: 5/8/2022
    24h volume: 17288.22990867702
    Total volume: 109982.0041565043

**Comment by llamatester** [comment]
The adapter at projects/arctic exports TVL: 
        
 
 
    aurora                    103.94 k
    
    total                    103.94 k

**Review by g1nt0ki — APPROVED** [review]

**Comment by g1nt0ki** [comment]
@LpcAries what is the relation between Izumi and Arctic team?

**Comment by LpcAries** [comment]
The Arctic project is co-incubated by Proximity Labs and iZUMi. The tech team of iZUMi provides Arctic with a complete AMM technical solution. @g1nt0ki
